test/unit/finance/bill_variable_for_plan_changed_test.rb

```diff
@@ -18,7 +18,7 @@ class Finance::BillVariableForPlanChangedTest < ActiveSupport::TestCase
   end
 
   test "bill for variable" do
-    Timecop.travel(15.days.ago) if Time.now.mday == 1 # rubocop:disable Rails/TimeZone we don't use timezones in billing
+    travel_to(15.days.ago) if Time.now.mday == 1 # rubocop:disable Rails/TimeZone we don't use timezones in billing
```

Comment:
do we travel back anywhere here? Same question about the rest of test cases here

Reply:
Only when we are in the first day of the month... I don't know why, but it seems these tests are time sensitive, so I'm leaving the time travel. I added a teardown to return time to normal after the tests, though.

Reply:
makes sense, thanks

I wonder, don't we have a global teardown `ActiveSupport::TestCase`? How was time restored with `Timecop`?

Reply:
Actually now I see `test/test_helper.rb`, so it seems like only tests that do not inherit from `ActiveSupport::TestCase` need in their own `teardown`. Sorry for not noticing this before. Could you check where it is really needed?

Reply:
I removed the teardown